src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -1171,6 +1172,16 @@ void Transformations::MainSnippets() {
         CPU_DISABLE_PASS_COMMON(snippetsManager, snippets::pass::ExtractReshapesFromMHA);
     }
 
+#if defined(OPENVINO_ARCH_X86_64)
+    const bool isMlpSeqSupported = true;
+#else
+    const bool isMlpSeqSupported = false;
```

Comment:
I suggest to add the same check for inference precision for `MlpSeq` - if we understand earlier can we tokenized MLP sequence, we can disable this pass - it will save time on model compilation stage. For example, if if inference precision is `f16` and isa is `avx2_vnni_2`, we understand that our `BrgemmCPU` doesn't support this configuration yet and it doesn't make sense to tokenize MLP.
So we can rename `is_infer_prc_supported_by_MHA` into `is_infer_prc_supported_by_brgemm` and reuse it for MHA and MLP

What do you think?

Reply:
I agree, applied